src/test/java/io/jenkins/plugins/analysis/warnings/ParsersITest.java

```diff
@@ -242,6 +242,15 @@ public void shouldFindAllJsHintIssues() {
         shouldFindIssuesOfTool(6, new JsHint(), "jshint.xml");
     }
 
+    /**
+     * Runs the JUnit parser on an output file that contains 2 and 1 issues.
+     */
+    @Test
+    public void shouldFindAllJUnitIssues() {
+        shouldFindIssuesOfTool(2,new JUnit(),"junit.xml");
+        shouldFindIssuesOfTool(1,new JUnit(),"TEST-org.jenkinsci.plugins.jvctb.perform.JvctbPerformerTest.xml");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' is not followed by whitespace.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4880215)